opencga-server/pom.xml

```diff
@@ -249,7 +368,7 @@
                 <activeByDefault>true</activeByDefault>
             </activation>
             <dependencies>
-                <dependency>
+                <!--<dependency>
```

Comment:
You can not remove this! This dependency is used by reflection, so the dependency analysis doesn't detect as directly used